### PR TITLE
Enable custom target result status

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
We are just reusing the functionality within updater package, which is already unit tested

### CODE REVIEW

**Review by michelle192837 — COMMENTED** [review]
This looks good, but I think this should include where we're adding customTargetStatus().

**Comment by sultan-duisenbay** [comment]
> This looks good, but I think this should include where we're adding customTargetStatus().

Will do

File: pkg/updater/resultstore/resultstore.go [hunk 1]
```diff
@@ -59,6 +63,12 @@ type singleActionResult struct {
 	ActionProto           *resultstorepb.Action
 }
 
+// make singleActionResult satisfy TargetResult interface
+func (sar *singleActionResult) TargetStatus() statuspb.TestStatus {
+	status := convertStatus[sar.TargetProto.GetStatusAttributes().GetStatus()]
```
Comment (chases2) [inline pkg/updater/resultstore/resultstore.go:68]: nit: one-line return would be just as clear

**Review by chases2 — APPROVED** [review]

File: pkg/updater/resultstore/resultstore.go (hunk 1 above)
Comment (chases2) [reply pkg/updater/resultstore/resultstore.go:68]: Oh no I forgot to hold :(

**Pull request merged**